packages/pds/service/index.js

```diff
@@ -84,12 +85,28 @@ const main = async () => {
   })
   const viewMaintainer = new ViewMaintainer(migrateDb)
   const viewMaintainerRunning = viewMaintainer.run()
+
+  const periodicModerationActionReversal = new PeriodicModerationActionReversal(
+    db,
+    pds.ctx,
+  )
+  const periodicModerationActionReversalRunning =
+    periodicModerationActionReversal.run()
+
   await pds.start()
   // Graceful shutdown (see also https://aws.amazon.com/blogs/containers/graceful-shutdowns-with-ecs/)
   process.on('SIGTERM', async () => {
     await pds.destroy()
+
+    // Gracefully shutdown view-maintainer
     viewMaintainer.destroy()
     await viewMaintainerRunning
+
+    // Gracefully shutdown periodic-moderation-action-reversal
+    periodicModerationActionReversal.destroy()
+    await periodicModerationActionReversalRunning
```

Comment:
We might consider calling this before `pds.destroy()`.  Since it uses the `db` which is closed by `pds`, it could start to fail if it's stopped before `pds`.

Reply:
I see so viewmaintainer uses migrateDb which is why we can wait for it to destroy after I guess?